pyuvdata/uvdata/uvdata.py

```diff
@@ -11173,6 +11510,14 @@ def read(
             combining multiple files, which would otherwise result in an error being
             raised because of attributes not matching. Doing so effectively adopts the
             name found in the first file read in. Default is False.
+        allow_flex_pol : bool
+            If only one polarization per spectral window is read (and the polarization
+            differs from window to window), allow for the `UVData` object to use
+            "flexible polarization", which compresses the polarization-axis of various
+            attributes to be of length 1, sets the `flex_spw_polarization_array`
+            attribute to define the polarization per spectral window. Only applicable
+            for MIR and MS filetypes, otherwise this argument is ignord. Default is
```

Comment:
Typo in ignored.

Reply:
Didn't mean to ignore that. Fixed in 5c36be1a.